src/friendly-iframe-embed.js

```diff
@@ -627,5 +627,6 @@ export function whenContentIniLoad(elementOrAmpDoc, hostWin, rect) {
  * @return {boolean}
  */
 export function isInFie(element) {
-  return !!closestAncestorElementBySelector(element, '.i-amphtml-fie');
+  return element.classList.contains('i-amphtml-fie') ||
```

Comment:
@zhouyx made this change so documentElement itself also passes the check

Reply:
👍 Sounds good